include/utils/CommonUtils.php

```diff
@@ -2699,6 +2699,27 @@ function getUItypeByFieldName($module, $fieldname) {
 	return $uitype;
 }
 
+/**
+ * Function to get the Type of data for a field by the fieldname.
+ * Takes the input as $module - module name,and fieldname of the field
+ * returns the type of data, varchar type
+ */
+function getTypeOfDataByFieldName($module, $fieldname) {
+	global $log, $adb;
+	$log->debug('> getTypeOfDataByFieldName ' . $module);
+	$tabIdList = array();
+	//To find tabid for this module
+	$tabIdList[] = getTabid($module);
+	if ($module == 'Calendar') {
+		$tabIdList[] = getTabid('Events');
+	}
+	$sql = 'select typeofdata from vtiger_field where tabid IN (' . generateQuestionMarks($tabIdList) . ') and fieldname=?';
+	$result = $adb->pquery($sql, array($tabIdList, $fieldname));
+	$tod = explode('~', $adb->query_result($result, 0, 'typeofdata'))[0];
```

Comment:
I would like you to save the result of the explode in an array and access the [0] element from there as we still have some clients on 5.6 and I don't remember when this syntax was introduced